### PR TITLE
fix: updates environment variable configuration

### PR DESCRIPTION
**Pull request by kelsos**
- Updating the runtime config a bit so that we can follow the [documentation](https://nuxt.com/docs/guide/going-further/runtime-config#example)
- Changes the proxy configuration so that we only need to pass the domain as a single env variable.

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -113,11 +121,11 @@ export default defineNuxtConfig({
...
       recaptcha: {
-        siteKey: process.env.RECAPTCHA_SITE_KEY,
+        siteKey: '',
       },
-      baseUrl: process.env.BASE_URL || '',
-      maintenance: process.env.MAINTENANCE || 'false',
-      testing: process.env.TESTING,
+      baseUrl: '',
+      maintenance: 'false',
+      testing: 'false',
```
Comment (lukicenturi) [inline nuxt.config.ts:128]: we can't use boolean instead here?

Comment (lukicenturi) [reply nuxt.config.ts:128]: Okay I see, it only accept string

**Review by lukicenturi — APPROVED** [review]
LGTM!

File: nuxt.config.ts (hunk 1 above)
Comment (kelsos) [reply nuxt.config.ts:128]: > Okay I see, it only accept string

Yes, I tried to put a boolean there in the beginning :'(

**Pull request merged**